docs/try-viam/rover-resources/rover-tutorial.md

```diff
@@ -255,10 +255,14 @@ The following are just a few ideas, but you can expand or modify the rover kit w
 - For GPS navigation, we support NMEA (using serial and I<sup>2</sup>C) and RTK.
   Make and model don't make a difference as long as you use these protocols.
   See [Movement Sensor Component](/components/movement-sensor/) for more information.
-- For [LiDAR laser range scanning](/services/slam/cartographer/), we recommend Velodyne, or RPLIDAR (including A1, which is a sub-$100 LIDAR).
+- For [LiDAR laser range scanning](/services/slam/cartographer/), we recommend Velodyne, or RPlidar (including A1, which is a sub-$100 LIDAR).
```

Comment:
Updating `RPlidar` caps throughout to match [Slamtec's convention](https://www.slamtec.ai/home/rplidar_a1/).

Reply:
might be good adding that to our vale rules...

Reply:
https://github.com/viamrobotics/docs/pull/1574

Reply:
Expanded to entire page.